examples/FromJim/spice.mzLib

```diff
@@ -0,0 +1,794 @@
+<mzSpecLib 1.0>
+MS:1003188|library name = spice
+MS:1003190|library version = 1.0
+MS:1001017|release date = (UTC) 5/25/2023 3:45:29 PM
+
+<Spectrum=1>
+MS:1003061|library spectrum name = JWH-250-5OH
+MS:1003062|library spectrum index = 795
+MS:1000894|retention time = 3.53813
+MS:1000864|chemical formula = 
+MS:1002894|InChiKey = 
+MS:1009100|CASNo = 
+MS:1000868|Smiles formula = 
+MS:1009101|compound class = 
+MS:1003059|number of peaks = 51
+<Analyte=1>
+MS:1000744|selected ion m/z = 352.19000
+MS:1000045|collision energy = 0
+MS:1000041|charge state = 1
+MS:1000008|ionization type=MS:1000073|electrosprary ionization
+MS:1000465|scan polarity=MS:1000130|positive scan
+<Peaks>
+51.0236 41921.14
+52.3338 18399.55
+63.9969 22028.22
+65.0390 263790.31
+67.0550 30223.43
+69.0705 504983.50
+74.5264 25105.37
+77.0390 117307.05
+78.0471 125893.67
+78.2767 24682.19
+85.0653 44129.83
+87.0808 124820.38
+91.0546 4938715.00
+92.0579 32281.12
+93.0703 4039066.50
+95.0497 91643.40
+103.0546 31422.36
+106.0417 61616.02
+107.0493 375355.88
+109.0650 2526102.50
+118.0650 245160.55
+119.0605 1306983.88
+121.0649 80814008.00
+122.0681 1211911.50
+129.6649 23031.13
+130.0651 2914473.00
+131.0491 2273469.50
+131.0699 34800.33
+132.0810 254987.64
+135.0337 46479.04
+135.0439 1112392.50
+144.0443 231014.09
+144.0805 1083481.13
+149.0595 141239.42
+158.0962 52338.03
+171.1041 48832.38
+186.1275 1500389.75
+204.1381 1171604.25
+216.1380 4790471.00
+217.1419 138852.80
+230.1172 1742078.38
+233.0834 39347.97
+246.0902 160523.73
+247.0991 116021.20
+248.1052 52735.21
+277.0875 24467.71
+302.1530 60953.01
+306.1864 41791.95
+319.1559 1327439.63
+334.1782 192063.28
+352.1904 138860.03
+
+<Spectrum=2>
+MS:1003061|library spectrum name = HU-210
+MS:1003062|library spectrum index = 1057
+MS:1000894|retention time = 4.63888
+MS:1000864|chemical formula = C25H38O3
+MS:1002894|InChiKey = 
+MS:1009100|CASNo = 

```

Comment:
If the value is unknown, just omit the attribute?

Reply:
No change made, since the specification has been updated.